src/commands/BootstrapCommand.js

```diff
@@ -28,16 +42,11 @@ export default class BootstrapCommand extends Command {
     this.progressBar.init(this.packages.length);
     this.logger.info("Linking all dependencies");
 
-    const ignore = this.flags.ignore || this.repository.bootstrapConfig.ignore;
-
-    // Get a filtered list of packages that will be bootstrapped.
-    const todoPackages = PackageUtilities.filterPackages(this.packages, ignore, true);
-
-    // Get a trimmed down graph that includes only those packages.
-    const filteredGraph = PackageUtilities.getPackageGraph(todoPackages);
+    // Get a trimmed down graph that includes only bootstrapping packages.
+    const filteredGraph = PackageUtilities.getPackageGraph(this.bootstrapPackages);
 
     // As packages are completed their names will go into this object.
-    const donePackages = {};
+    this.donePackages = {};
```

Comment:
want to keep track of bootstrapped packages within the class to display the number at the end, see L#35.